script/Cargo.lock

```diff
@@ -0,0 +1,1858 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I thought adding `script` into the workspace means we have one single Cargo.lock for the entire workspace. Is that not the case? What happens if we simply remove this file?

Reply:
Yeah Cargo.lock inside script is no longer required and should be removed.